etc/ci/performance/runner.py

```diff
@@ -116,6 +116,7 @@ def valid_timing(timing, testcase=None):
         if (timing is None or
                 testcase is None or
                 timing.get('title') == 'Error response' or
```

Comment:
I think we can get rid of this case; I don't see any code that would generate a page with that title.

Reply:
Done.